Common/cpp/headers/NativeModules/NativeReanimatedModule.h

```diff
@@ -59,13 +59,13 @@ class NativeReanimatedModule : public NativeReanimatedModuleSpec {
     void render();
     void onEvent(std::string eventName, std::string eventAsString);
 
+    std::unique_ptr<jsi::Runtime> runtime;
```

Comment:
https://stackoverflow.com/questions/2254263/order-of-member-constructor-and-destructor-calls